src/modules/visitor-space/components/MaintainerFilterForm/MaintainerFilterForm.tsx

```diff
@@ -0,0 +1,109 @@
+import { yupResolver } from '@hookform/resolvers/yup';
+import { CheckboxList } from '@meemoo/react-components';
+import clsx from 'clsx';
+import { compact, without } from 'lodash';
+import { FC, useEffect, useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { useSelector } from 'react-redux';
+import { useQueryParams } from 'use-query-params';
+
+import { SearchBar } from '@shared/components';
+import useTranslation from '@shared/hooks/use-translation/use-translation';
+import { selectIeObjectsFilterOptions } from '@shared/store/ie-objects';
+import { visitorSpaceLabelKeys } from '@visitor-space/const';
+import { VisitorSpaceFilterId } from '@visitor-space/types';
+
+import {
+	MAINTAINER_FILTER_FORM_QUERY_PARAM_CONFIG,
+	MAINTAINER_FILTER_FORM_SCHEMA,
+} from './MaintainerFilterForm.const';
+import { MaintainerFilterFormProps, MaintainerFilterFormState } from './MaintainerFilterForm.types';
+
+const defaultValues = {
+	maintainers: [],
+};
+
+const MaintainerFilterForm: FC<MaintainerFilterFormProps> = ({ children, className }) => {
+	const { tHtml, tText } = useTranslation();
+
+	const [query] = useQueryParams(MAINTAINER_FILTER_FORM_QUERY_PARAM_CONFIG);
+	const [search, setSearch] = useState<string>('');
+	const [shouldReset, setShouldReset] = useState<boolean>(false);
+	const [selection, setSelection] = useState<string[]>(() =>
+		compact(query[VisitorSpaceFilterId.Maintainers] || [])
+	);
+
+	const { setValue, reset, handleSubmit } = useForm<MaintainerFilterFormState>({
+		resolver: yupResolver(MAINTAINER_FILTER_FORM_SCHEMA()),
+		defaultValues,
+	});
+
+	const buckets = (
+		useSelector(selectIeObjectsFilterOptions)?.schema_maintainer?.schema_identifier?.buckets ||
```

Comment:
The backend has been fixed to be able to return the maintainers:
![image](https://user-images.githubusercontent.com/1710840/227906705-03082f1f-500f-4402-bc64-fd68d5443e1b.png)

Kan ook in aparte PR: https://meemoo.atlassian.net/browse/ARC-1510

```suggestion
		useSelector(selectIeObjectsFilterOptions)?.['schema_maintainer.schema_name']?.buckets ||
```

Reply:
Ik pik het mee op in deze PR ;)